cms/static/js/views/modals/course_outline_modals.js

```diff
@@ -488,6 +475,52 @@ define(['jquery', 'backbone', 'underscore', 'gettext', 'js/views/baseview',
             };
         }
     });
+    AccessEditor = AbstractEditor.extend({
+        templateName: 'access-editor',
+        className: 'edit-settings-access',
+        events : {
+            'change #prereq': 'handlePrereqSelect',
+            'keyup #prereq_min_score': 'validate'
+        },
+        afterRender: function () {
+            AbstractEditor.prototype.afterRender.call(this);
+            var prereq = this.model.get('prereq') || '';
+            var prereq_min_score = this.model.get('prereq_min_score') || '';
+            this.$('#is_prereq').prop('checked', this.model.get('is_prereq'));
+            this.$('#prereq option[value="' + prereq + '"]').prop('selected', true);
+            this.$('#prereq_min_score').val(prereq_min_score);
+            this.$('#prereq_min_score_input').toggle(prereq.length > 0);
+        },
+        handlePrereqSelect: function () {
+            var showPrereqInput = this.$('#prereq option:selected').val().length > 0;
+            this.$('#prereq_min_score_input').toggle(showPrereqInput);
+        },
+        validate: function () {
```

Comment:
nit: A more specific name for the validator method since we're validating just the required min score here. e.g. validateScore